doc/man7/EVP_KDF_SS.pod

```diff
@@ -0,0 +1,224 @@
+=pod
+
+=head1 NAME
+
+EVP_KDF_SS - The Single Step EVP_KDF implementation. (Also known as One
+-Step Key Derivation).
```

Comment:
That's an odd line break.  Also, I'm not sure about a parenthesis here, so if you want both variants in the title (and resulting apropos), may I suggest this?

```
EVP_KDF_SS - The Single Step / One-Step EVP_KDF implementation
```

(also, I think its customary for these title not to end with a period)

If you want, you can dive a little bit into the different names in the beginning of the DESCRIPTION.